src/Main.tsx

```diff
@@ -158,7 +186,15 @@ const App: React.FunctionComponent = () => {
                     style={{ overflowY: "auto" }}
                     data-bind="if: isAuthWithResourceToken(), react:resourceTreeForResourceToken"
                   />
-                  <div style={{ overflowY: "auto" }} data-bind="if: !isAuthWithResourceToken(), react:resourceTree" />
+                  <div style={{ overflowY: "auto" }} data-bind="if: !isAuthWithResourceToken()">
+                    <ResourceTree
+                      explorer={explorer}
+                      lastRefreshedTime={lastRefreshTime}
+                      galleryContentRoot={galleryContentRoot}
+                      myNotebooksContentRoot={myNotebooksContentRoot}
+                      gitHubNotebooksContentRoot={gitHubNotebooksContentRoot}
+                    />
+                  </div>
```

Comment:
```suggestion
                  <div style={{ overflowY: "auto" }}>
                    isAuthWithResourceToken() || <ResourceTree
                      explorer={explorer}
                      lastRefreshedTime={lastRefreshTime}
                      galleryContentRoot={galleryContentRoot}
                      myNotebooksContentRoot={myNotebooksContentRoot}
                      gitHubNotebooksContentRoot={gitHubNotebooksContentRoot}
                    />
                  </div>
```

Reply:
Is this an opportunity to remove one more knockout binding?

Reply:
It's a bit more involved as `isAuthWithResourceToken` isn't available in `Main.tsx`, but in `Explorer` (btw, you probably meant to enclose this expression inside `{{ }}`). So we'll first move everything to react, then will come back to remove the ko's (ie replace them as some state in `Main.tsx`).